libs/ui-lib/lib/common/components/ui/formik/LabelField.tsx

```diff
@@ -44,64 +50,54 @@ export const LabelField: React.FC<LabelFieldProps> = ({
-          labelIcon={labelIcon}
-        >
-          {t(
-            "ai:Enter key=value and then press 'enter' or 'space' or use a ',' to input the label.",
-          )}
-          <div className="co-search-input pf-c-form-control">
-            <TagsInput
-              {...field}
-              onChange={(tags) => {
-                setValue(tags);
-                setInput('');
-                onChange && onChange(tags);
-                !touched && setTouched(true);
-              }}
-              addKeys={[13, 32, 188]}
-              renderTag={({ tag, key, onRemove, getTagDisplayValue }) => (
-                <LabelValue
-                  key={key as number}
-                  onClose={() => onRemove(key as number)}
-                  value={getTagDisplayValue(tag)}
-                />
-              )}
-              addOnBlur
-              inputProps={{
-                autoFocus: false,
-                className: 'label-field__input',
-                // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-                placeholder: field.value?.length ? '' : 'app=frontend',
-                spellCheck: 'false',
-                id: 'tags-input',
-                value: input,
-                // eslint-disable-next-line
-                onChange: (e: any) => setInput(e.target.value),
-                ['data-test']: 'tags-input',
-              }}
+    <FormGroup fieldId={fieldId} label={label} isRequired={isRequired} labelIcon={labelIcon}>
+      {t("ai:Enter key=value and then press 'enter' or 'space' or use a ',' to input the label.")}
+      <div className="co-search-input pf-v5-c-form-control">
+        <TagsInput
+          {...field}
+          onChange={(tags) => {
+            setValue(tags);
+            setInput('');
+            onChange && onChange(tags);
+            !touched && setTouched(true);
+          }}
+          addKeys={[13, 32, 188]}
+          renderTag={({ tag, key, onRemove, getTagDisplayValue }) => (
+            <LabelValue
+              key={key as number}
+              onClose={() => onRemove(key as number)}
+              value={getTagDisplayValue(tag)}
             />
-          </div>
-        </FormGroup>
-      </StackItem>
-      <StackItem>
-        {errorMessage && (
-          <HelperText fieldId={fieldId} isError>
-            {errorMessage}
+          )}
+          addOnBlur
+          inputProps={{
+            autoFocus: false,
+            className: 'label-field__input',
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+            placeholder: field.value?.length ? '' : 'app=frontend',
+            spellCheck: 'false',
+            id: 'tags-input',
+            value: input,
+            // eslint-disable-next-line
+            onChange: (e: any) => setInput(e.target.value),
+            ['data-test']: 'tags-input',
+          }}
+        />
+      </div>
+      {(errorMessage || helperText) && (
```

Comment:
Same as before regarding this repeating pattern.

Reply:
https://issues.redhat.com/browse/MGMT-16680